### PR TITLE
Add section on how to build plugin to README

### PR DESCRIPTION
**Pull request by dkoontz**
Small section giving the steps needed to compile locally and replace the standard plugin with the local version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,3 +116,15 @@ You may be able to get away without thinking about all this if your project spec
...
 is no visual indication, currently this will basically be for 'import' warnings,
 and can be triggered by 'alt-enter' (PureScript: Show Quick Fixes).
+
+## Hacking on atom-ide-purescript
+
+Install dependencies
+  • `bower install`
+  • `npm install`
+
+Build: `pulp build`
```
Comment (nwolverson) [inline README.md]: I'd remove this. Just need to run one step `npm install` or `npm run bundle`. (or as I prefer `npm run -s bundle`).

Comment (dkoontz) [inline README.md]: So `npm install` will also trigger `bower install` or are you saying that when you run `npm run bundle` that will do the bower/npm installation?

Comment (nwolverson) [inline README.md]: No, remove `pulp build` only. `npm install` will actually trigger `npm run bundle`, which is all you need to build each time (it includes `pulp build` as part of a command line), but usually I'd run that directly.

**Comment by nwolverson** [comment]
Merged in de4f26a34921a10b45a24210cf23a90d70f7cc57 (fixed your `plup build` :D)

**Pull request closed without merging**